tasks/main.yml

```diff
@@ -34,3 +35,8 @@
     state: stopped
     enabled: no
   when: not ntp_enabled
+
+- name: Generate ntp.conf file
+  template: src=ntp.conf.j2 dest=/etc/ntp.conf
+  notify: restart ntp
+  when: ntp_enabled
```

Comment:
To conform to the comments I made in the PR thread, this should be something like:

```
  when: ntp_servers is defined and ntp_servers
```

However, I'm thinking more comprehensively—we might want to add a `ntp_managed_config` default, default it to `false`, and use that as the flag as to whether to manage the config file.

I also need to verify that the `/etc/ntp.conf` file is the same for Debian/Ubuntu/CentOS/RHEL, or if we need a custom variable here for cross-platform compatibility.